pkg/distro/image_config.go

```diff
@@ -110,6 +111,10 @@ type ImageConfig struct {
 	MountUnits *bool
 }
 
+type WSLConfig struct {
+	BootSystemd bool
+}
```

Comment:
This is probably the first instance of a substruct of ImageConfig being defined here and it makes me think we need to figure out a more general strategy.

See my main comment.